pkg/distro/image_config.go

```diff
@@ -89,7 +89,8 @@ type ImageConfig struct {
 
 	WSL *wsl.WSL `yaml:"wsl,omitempty"`
 
-	Users []users.User
+	Users    []users.User
+	LockRoot *bool `yaml:"lock_root,omitempty"`
```

Comment:
We have a `LockRootUser` option a few lines down (line 121 in this file) - how are the two related? And could we maybe reuse/unify the names (as they sound so similar)?